File: SyntaxPractice/problem-1/orchestrator/Main.py
import os
import sys

sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))

from logging.Logger import Logger
from service.Service import Service

Logger.info(
    "Application Started"
)

n = int(
    input("Enter number of elements: ")
)

print(
    f"Enter {n} integers:"
)

numbers = []

for _ in range(n):
    numbers.append(
        int(input())
    )

service = Service()

result = service.process(
    numbers
)

print()
print(result)

Logger.info(
    "Application Completed"
)

File: SyntaxPractice/problem-1/orchestrator/Main.java
import logging.Logger;
import model.Result;
import service.Service;

import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Logger.info("Application Started");

        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter number of elements: ");

        int n = scanner.nextInt();

        List<Integer> numbers = new ArrayList<>();

        System.out.println(
                "Enter " + n + " integers:"
        );

        for (int i = 0; i < n; i++) {
            numbers.add(
                    scanner.nextInt()
            );
        }

        Service service = new Service();

        Result result =
                service.process(
                        numbers
                );

        System.out.println();
        System.out.println(result);

        Logger.info(
                "Application Completed"
        );
    }
}
